src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -12,11 +15,17 @@ public class VisitorMessageService {
 
     private final VisitorMessageRepository visitorMessageRepository;
     private final EmailSender emailSender;
+    private final ReceiverService receiverService;
 
     public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreateDto) {
         VisitorMessage visitorMessage = VisitorMessageMapper.toModel(visitorMessageCreateDto);
         VisitorMessage savedVisitorMessage = visitorMessageRepository.save(visitorMessage);
-        emailSender.sendEmail("HARDCODED-EMAIL@EMAIL.PL", "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
+        JSONArray recipients = new JSONArray();
+        for (String email : receiverService.getAllReceivers().receivers()) {
+            recipients.put(new JSONObject().put("Email", email).put("Name", email));
+        }
+        emailSender.sendEmails(recipients, "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
```

Comment:
KARUN VISTOR MESSAGE -> KARUNA VISTOR MESSAGE i misspell it in last changes i think